eq-author-api/docker-compose.yml

```diff
@@ -5,6 +5,7 @@ services:
       context: .
     depends_on:
       - db
```

Comment:
I'd like to see the PR that drops postgres ASAP

Reply:
I think there is some other stuff still using the DB so this needs to be changed 1st